### PR TITLE
Add CSharp examples for WebDriver

### PR DESCRIPTION
**Pull request by imba-tjd**
Align with Java code.

### CODE REVIEW

File: wd.html [hunk 1]
```diff
@@ -281,6 +281,7 @@ <h4>Chromium/Chrome</h4>
 <pre>
  <code class=java>System.setProperty("webdriver.chrome.driver", "/path/to/chromedriver");</code>
  <code class=python>Chrome(executable_path='/path/to/chromedriver')</code>
+ <code class=cs>new ChromeDriver("/path/to/chromedriver")</code>
```
Comment (TheTestLynx) [inline wd.html]: Missing ;

File: wd.html [hunk 2]
```diff
@@ -739,7 +743,7 @@ <h3>Switching windows or tabs</h3>
     # Wait for the new tab to finish loading content
     wait.until(EC.title_is("Selenium documentation"))</code>
  <code class=cs>//Store the ID of the original window
-String originalWindow = driver.CurrentWindowHandle;
+string originalWindow = driver.CurrentWindowHandle;
```
Comment (TheTestLynx) [inline wd.html:748]: arguable :)

File: wd.html [hunk 3]
```diff
@@ -1061,15 +1100,25 @@ <h3>Get Window Position</h3>
 position = driver.get_window_position()
 x1 = position.get('x')
 y1 = position.get('y')</code>
+ <code class=cs>//Access each dimension individually
+int x = driver.Manage().Window.Position.X;;
```
Comment (TheTestLynx) [inline wd.html]: double ; at the end

File: wd.html (hunk 2 above)
Comment (imba-tjd) [reply wd.html:748]: I didn't read selenium's source code, but MS's libraries and docs prefer `string` than `String`.

File: wd.html (hunk 1 above)
Comment (imba-tjd) [inline wd.html]:
```suggestion
 <code class=cs>new ChromeDriver("/path/to/chromedriver");</code>
```

File: wd.html (hunk 3 above)
Comment (imba-tjd) [inline wd.html]:
```suggestion
int x = driver.Manage().Window.Position.X;
```

File: wd.html (hunk 2 above)
Comment (TheTestLynx) [reply wd.html:748]: Sure

**Review by TheTestLynx — APPROVED** [review]
Good to go, thanks!

**Review by diemol — APPROVED** [review]
Thanks @imba-tjd!

**Pull request merged**